docs/api/text/tsne.rst

```diff
@@ -20,20 +20,30 @@ After importing the required tools, we can :doc:`load the corpus <corpus>` and v
     corpus = load_corpus('hobbies')
     tfidf  = TfidfVectorizer()
 
-    docs   = tfidf.fit_transform(corpus.data)
-    labels = corpus.target
+    X   = tfidf.fit_transform(corpus.data)
+    y   = corpus.target
```

Comment:
Thank you for making this change! Please note that on Line 53 below and Line 66 below, `docs` needs to be updated to `X` because of this change (in the clustering section).

Reply:
Sure,I am going to change